app/models/template_kind.rb

```diff
@@ -22,7 +22,8 @@ def self.default_template_labels
       "ZTP" => N_("ZTP PXE template"),
       "POAP" => N_("POAP PXE template"),
       "cloud-init" => N_("Cloud-init template"),
-      "registration" => N_("Registration template"),
+      "global_registration" => N_("Global registration template"),
```

Comment:
I don't know, doesn't sound much more descriptive to me IMO. I would say that renaming `registration` template to `host_setup` would be way more better, since we don't do any registration at all there.

Reply:
I'm tempted to say registration bootstrap, but bootstrap is used already :-) Therefore the init sounded better to me. `host_setup` evokes it's host related or contains setup specific for the host, which it does not at this point. The results is the script that has instructions for how to initiate the registration for the host. But the template for such script is generic (global). We can come up with something not overloaded yet, such as foreman_init, foreman_bootstrap.

Reply:
after quick chat:
registration -> host_initial_configuration
global_registration -> registration (remains)